public/css/style.css

```diff
@@ -436,15 +473,15 @@ select#bottom_language_selector {
 
 .comment-date {
 	font-size: smaller;
-	width: auto; /* Undo bootstrap's fixed width */
+	/* Undo bootstrap's fixed width */
```

Comment:
I prefer end of line comments with // and new line comments with /* */. So I'm more for his way of doing it